### ISSUE TITLE
Documentation for detection needs updates

### ISSUE THREAD

**Issue by edublancas** [paninski-lab/yass#143]
AFAIK, @jinhyunglee updated the detection methods so they now return clear spikes and all spikes but the docs still say:

```

    Returns
    -------
    clear_scores: numpy.ndarray (n_spikes, n_features, n_channels)
        3D array with the scores for the clear spikes, first simension is
        the number of spikes, second is the nymber of features and third the
        number of channels

    spike_index_clear: numpy.ndarray (n_clear_spikes, 2)
        2D array with indexes for clear spikes, first column contains the
        spike location in the recording and the second the main channel
        (channel whose amplitude is maximum)

    spike_index_collision: numpy.ndarray (n_collided_spikes, 2)
        2D array with indexes for collided spikes, first column contains the
        spike location in the recording and the second the main channel
        (channel whose amplitude is maximum)
```

We need to update that, right?

**Comment by jinhyunglee** [comment]
Yes. I will update soon

**Comment by edublancas** [comment]
Fixed in #200
